aten/src/ATen/native/native_functions.yaml

```diff
@@ -4634,6 +4635,18 @@
 
 - func: trapz.dx(Tensor y, *, float dx=1, int dim=-1) -> Tensor
 
+- func: _nested_from_padded(Tensor padded, Tensor cpu_nested_shape_example, bool fuse_transform_0213=False) -> Tensor
+  device_check: NoCheck # cpu_nested_shape_example will always be on CPU
+  dispatch:
+    CPU: nested_from_padded_generic
```

Comment:
dumb question is the "generic" implementation generic meaning that it would potentially work for other backends aside from CPU? Or the implementation cpu-specific.

If it's actually generic, and there's any value at all in having nested tensor work with other backends like XLA, then you could mark it `CompositeExplicitAutograd`.

Reply:
it appears to actually be generic, but I will not be able to support it on arbitrary backends, so I've erred on the side of only setting up what I need for transformers.